src/poliastro/core/perturbations.py

```diff
@@ -217,6 +217,32 @@ def third_body(t0, state, k, k_third, perturbation_body):
     k : float
         gravitational constant, (km^3/s^2)
     perturbation_body: a callable object returning the position of the pertubation body that causes the perturbation
+        .. py:function:: perturbation_body() -> position_of_perturbation_body
+        .. py:class:: Perturbation_body() -> position_of_perturbation_body
```

Comment:
I don't understand what are these `Perturbation_body` and `Star` classes :/

Reply:
These can be seen from this [link](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-signatures) ... The normal `function` directive is for representing a function in the documentation for the type expected or more formally, knowing the function signature. I tried to implement a similar thing with the `class` directive so that examples of function & class in callables be stated. 
This is how that `py:function` is represented 
![image](https://user-images.githubusercontent.com/66117751/107869709-e92bd800-6eb6-11eb-9b41-5968f1e72c3b.png)
This is how that `py:class` is represented 
![image](https://user-images.githubusercontent.com/66117751/107869719-f779f400-6eb6-11eb-8c45-c4230f4e73fb.png)

Reply:
Any updates for your side on how we should proceed so that it becomes more beginner-friendly documentation?